File: app/src/main/java/com/example/cricscore/MainActivity.java
package com.example.cricscore;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    Button startScoring;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        startScoring = (Button) findViewById(R.id.startBtn);

        startScoring.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, ScoreDetails.class));

            }
        });

    }
}

File: app/src/main/java/com/example/cricscore/MainActivity.kt
package com.example.cricscore

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import android.widget.Button
import com.example.cricscore.ScoreDetails as ScoreDetails1

class MainActivity : AppCompatActivity() {
    lateinit var startScoring: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        startScoring = findViewById(R.id.startBtn)

        startScoring.setOnClickListener { startActivity(Intent(this@MainActivity, ScoreDetails1::class.java)) }

    }
}